### DISCUSSION TITLE
Does sensor node hardware require two ethernet cables

### DISCUSSION THREAD

**Discussion by kunalhiremath** [Security-Onion-Solutions/securityonion#9740] [Unsupported Versions]
I want to configure sesnor node on a tower server, should I need to connect two ethernet cables to get two interfaces. One interface for management interface and another interface for monitoring.

**Comment by innovate-support** [comment]
I would say yes, unless you only want to use the sensor for wazuh/ossec type logs. The second nic is used to ingest the network traffic it sees from your mirrored switch port or tap. I attempted to use a USB network adapter to try to pull in traffic logs, but it never worked. Physical second nic is the way to go.

**Reply by kunalhiremath** [reply]
Thank you

**Reply by kunalhiremath** [reply]
hey @innovate-support I had one more doubt, before sensor node configuration I can see two IP addresses but after sensor node is setup and when I restart the machine I can see that the IP address for the second interface will disappear. Why is that? Is not the interface that is set as monitor interface when setting up the sensor node!

**Comment by cm-ops** [comment]
_If you plan to sniff network traffic from a tap or span port, then you will need one or more interfaces dedicated to sniffing (no IP address). The installer will automatically disable NIC offloading functions such as tso, gso, and gro on sniffing interfaces to ensure that [Suricata](https://docs.securityonion.net/en/latest/suricata.html#suricata) and [Zeek](https://docs.securityonion.net/en/latest/zeek.html#zeek) get an accurate view of the traffic._ From https://docs.securityonion.net/en/latest/hardware.html?nic#nic

**Reply by kunalhiremath** [reply]
Thanks
